### PR TITLE
Update sane doc reports automation

### PR DESCRIPTION
**Pull request by agamm**
**This addresses some crucial changes to the DOCX reports, merging before Sunday would be awesome** 

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19429
fixes: https://github.com/demisto/etc/issues/19306

## Description
Updating the `sane-doc-reports` tag.
Docx reports will gracefully fail (inserting the error into the docx) while retaining the other elements that did not cause the failure. Plus added `itemsSection` element (heavily used in the investigation reports).

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [X] Code Review

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/fe2acff5a7beaab9f4d441a8f8348c822bce4f90/Scripts/SaneDocReport/CHANGELOG.md)

### CODE REVIEW

File: Scripts/SaneDocReport/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+- DOCX reports now fail gracefully (error shown in the docx instead of failure to generate completely).
+- Added the itemsSection element to DOCX reports (mainly used in investigation reports).
```
Comment (yaakovi) [inline Scripts/SaneDocReport/CHANGELOG.md]: where is the relevant changes for this? i only see the error handling.

Comment (agamm) [reply Scripts/SaneDocReport/CHANGELOG.md]: It is the internal docker (the tag change...)

**Comment by kirbles19** [comment]
@agamm Done.

**Comment by agamm** [comment]
@yaakovi ?

**Pull request merged**